ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -281,10 +281,25 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       OCP_ARCH: arm64
-      TEST_SCENARIOS: MCO|Cluster_Infrastructure
+      TEST_SCENARIOS: Cluster_Infrastructure
     test:
     - chain: openshift-e2e-test-qe-longrun
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-f28-longduration-mco-all
+  cron: 11 2 4,24 * *
```

Comment:
the cron frequency is inconsistent with the f28 in the job name, see here https://github.com/openshift/release/tree/master/ci-operator/config/openshift/openshift-tests-private#job-frequency
and also below new added arm64 jobs, others look good to me.